Tests/test_imagegrab.py

```diff
@@ -64,9 +64,13 @@ def test_grabclipboard(self):
             )
             p.communicate()
         else:
-            with pytest.raises(NotImplementedError) as e:
-                ImageGrab.grabclipboard()
-            assert str(e.value) == "ImageGrab.grabclipboard() is macOS and Windows only"
+            if not shutil.which("wl-paste"):
+                with pytest.raises(NotImplementedError) as e:
+                    ImageGrab.grabclipboard()
+                assert (
+                    str(e.value) == "wl-paste or xclip is required"
+                    " for ImageGrab.grabclipboard() on Linux"
+                )
```

Comment:
Can we use `pytest.raises(ValueError, match='...')`?

```suggestion
                with pytest.raises(
                    NotImplementedError,
                    match="wl-paste or xclip is required "
                    "for ImageGrab.grabclipboard() on Linux",
                ) as e:
                    ImageGrab.grabclipboard()
```

https://docs.pytest.org/en/7.1.x/reference/reference.html#pytest.raises

Reply:
Sure, I've added a commit for that.